packages/chaire-lib-backend/src/models/db/__tests__/users.db.test.ts

```diff
@@ -32,6 +32,15 @@ const user2 = {
     password_reset_token: 'sometokenforpasswordreset'
 };
 
+const userCaps = {
+    id: 3,
+    uuid: uuidV4(),
+    username: 'testCaps',
+    email: 'TEST@TEST.ORG',
```

Comment:
To make sure this is always valid, you can use user.email.toUpperCase(), in case one gets edited, and the other not. Even better, you could extract test@test.org to a variable and user toLowerCase() for user and toUpperCase for userCaps